### PR TITLE
Chore: cleanup & 0.58.4

### PR DESCRIPTION
**Pull request by kelset**
* fix the wordiness of the README
* tweak the release notes for all 0.58.x versions
* add the changelog for 0.58.4

### CODE REVIEW

**Comment by turnrye** [comment]
Thanks for tackling this @kelset! Left inline feedback. Generally watch for consistency too (capitalizing first letter of entries, not including MR number out of context).

File: CHANGELOG.md [hunk 1]
```diff
@@ -118,6 +161,10 @@ Thanks to those who gave feedback on our release candidates. If you're intereste
 - Fix an issue where **CameraRoll** wasn't showing the front-facing camera consistently during capture and preview ([4aeea4d](https://github.com/facebook/react-native/commit/4aeea4d))
 - Fix case where inline view is visible even though it should have been truncated ([70826db](https://github.com/facebook/react-native/commit/70826db) by [@rigdern](https://github.com/rigdern))
 
+### Known issues
+
+It is possible you face an AAPT error regarding missing resources, [here](https://github.com/infinitered/ignite-andross/issues/244) is an example of such error, in that case you should try to update the build tools versions to `buildToolsVersion = "28.0.2"` in your android/build.gradle file. If you maintain a react-native library which uses native code you should avoid using hardcoded versions of the build tools and use the packaged version numbers, here is an example you can [follow](https://github.com/react-native-community/react-native-linear-gradient/blob/master/android/build.gradle)
```
Comment (Salakar) [inline CHANGELOG.md]:
```suggestion
It is possible that you'll face an AAPT error regarding missing resources, [here](https://github.com/infinitered/ignite-andross/issues/244) is an example of this error, in this case, you should try to update the build tools versions to `buildToolsVersion = "28.0.2"` in your android/build.gradle file. If you maintain a react-native library which uses native code you should avoid using hardcoded versions of the build tools and use the packaged version numbers, here is an example you can [follow](https://github.com/react-native-community/react-native-linear-gradient/blob/master/android/build.gradle)
```

File: README.md [hunk 2]
```diff
@@ -2,19 +2,25 @@
...
 
-Progress towards release is tracked in issues. Look for these issues labelled _release status_. Soon a standardized format for the update will be published; please be patient while this settles.
+Progress towards release is tracked in issues, based on labels:
 
-Note that these issues are locked to keep conversation focused strictly on the status. Questions about the release should be directed to the [React Native](https://github.com/facebook/react-native/) repository or other communications channels.
+- _release status_: here you can read the status for the future/RC level releases
+- _stable_: here you can read the status for the current stable release
+- _backport requests_: here you can request cherry-picks for the current stable release, that will be included in the next patch version
+
+Note that these issues are made to keep conversation focused strictly on the status of each one - please refrain from going OT.
```
Comment (Salakar) [inline README.md]:
```suggestion
Note that these issues are made to keep the conversation focused strictly on the status of each one - please refrain from going off-topic.
```

File: README.md [hunk 3]
```diff
@@ -2,19 +2,25 @@
...
 
 ## Changelog
 
-The changelog in this repository is a community-provided effort to provide a helpful and informative summary of React Native's rapidly changing codebase. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and emojii use is roughly based on [gitmoji](https://gitmoji.carloscuesta.me/) recommendations.
+The changelog in this repository is a community-provided effort to provide a helpful and informative summary of React Native's rapidly changing codebase along with the full list of commits.
```
Comment (Salakar) [inline README.md]:
```suggestion
The changelog in this repository is a community-driven effort to provide a helpful and informative summary of React Native's rapidly changing codebase along with a full list of commits.
```

**Pull request merged**